DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,178 @@
+- [Developer Guide](#developer-guide)
+    - [Getting Started](#getting-started)
+        - [Git Clone logstash-output-opensearch](#git-clone-logstash-output-opensearch-repo)
+        - [Install Prerequisites](#install-prerequisites)
+            - [JRuby](#JRuby)
+            - [Docker](#docker)
+        - [Run Tests](#run-tests)
+        - [Run plugin](#run-plugin-in-logstash)
+        - [Configuration for Logstash Output OpenSearch Plugin](#configuration-for-logstash-output-opensearch-plugin)
+    - [Submitting Changes](#submitting-changes)
+
+# Developer Guide
+
+So you want to contribute code to logstash-output-opensearch? Excellent! We're glad you're here. Here's what you need to do.
+
+## Getting Started
+
+### Git Clone logstash-output-opensearch Repo
+
+Fork [opensearch-project/logstash-output-opensearch](https://github.com/opensearch-project/logstash-output-opensearch) and clone locally, e.g. `git clone https://github.com/[your username]/logstash-output-opensearch.git`.
+
+### Install Prerequisites
+
+#### JRuby
+
+This plugin builds using JRuby. This means you'll need JRuby with the Bundler gem installed.
+
+One easy way to get JRuby on is to use [rvm](https://rvm.io/rvm). Please check [here](https://rvm.io/rvm/install) on how to install rvm on your system. 
+If you use [rvm](https://rvm.io/rvm), then installing JRuby is also a piece of cake:
+```bash
+#Installing jruby
+rvm install jruby
+
+#Installing bundler
+jruby -S gem install bundler 
```

Comment:
I think this should be `rvm use jruby`, then `gem install bundler`.

Reply:
Ack